### PR TITLE
Add New Doc: Integrate MeterSphere into Pipelines

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

File: content/zh/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+   ![select-metersphere](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/select-metersphere.PNG)
+
+6. Input the MeterSphere API Keys created in the above step and the API Endpoint `http://NodeIP:NodePort`. You can set the rest values from the drop-down lists and then click **Generate Pipeline Script**.
+
+   ![input-values](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/input-values.PNG)
+
+   {{< notice note >}}
+
+   This tutorial uses the workspace `demo-workspace` and the project `demo` for demonstration purposes. You can create workspaces and projects and configure test cases on MeterSphere based on you needs. For more information about how to use MeterSphere, refer to [MeterSphere GitHub website](https://github.com/metersphere/metersphere/blob/master/README-EN.md).
```
Comment (Sherlock113) [inline content/zh/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md:91]: You can create workspaces and projects and configure test cases on MeterSphere based on you needs.
->
You need to create workspaces and projects, and configure test cases on MeterSphere first, or there will not be available items from the drop-down lists above.

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Felixnoo** [comment]
@Sherlock113 I added one more notice/note in line 137-141.

File: content/en/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+   ![select-metersphere](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/select-metersphere.PNG)
+
+6. Input the MeterSphere API Keys created in the above step and the API Endpoint `http://NodeIP:NodePort`. You can set the rest values from the drop-down lists and then click **Generate Pipeline Script**.
+
+   ![input-values](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/input-values.PNG)
+
+   {{< notice note >}}
+
+   This tutorial uses the workspace `demo-workspace` and the project `demo` from MeterSphere for demonstration purposes. You need to create workspaces and projects, and configure test cases on MeterSphere first, or there will not be available items from the drop-down lists shown in above image. For more information about how to use MeterSphere, refer to [MeterSphere GitHub website](https://github.com/metersphere/metersphere/blob/master/README-EN.md).
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md:91]: in above image
->
in the above image

File: content/en/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+3. Click **Show** under **Secret Key** to copy the Secret Key.
+
+   ![click-show](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/click-show.PNG)
+
+   ![copy-secret-key](/images/docs/devops-user-guide/tool-integration/integrate-metersphere/copy-secret-key.PNG)
+
+### Step 2: Install MeterSphere plugin on Jenkins
+
+1. Log in to the Jenkins console and click **Manage Jenkins**. For more information about how to log in to Jenkins console, refer to [Integrate SonarQube into Pipelines](../sonarqube/#step-5-add-the-sonarqube-server-to-jenkins).
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-integrate/integrate-metersphere.md:37]: Jenkins console
->
the Jenkins console

**Review by Sherlock113 — COMMENTED** [review]
Pay attention to the article.

**Pull request merged**